lib/Site.js

```diff
@@ -158,7 +171,7 @@ Site.initSite = function (rootPath) {
         if (fs.existsSync(indexPath)) {
           return Promise.resolve();
         }
-        return fs.outputFileAsync(indexPath, INDEX_MARKDOWN_DEFAULT);
+        return fs.outputFileAsync(indexPath, `${INDEX_MARKDOWN_DEFAULT}\n<include src="${FOOTER_FILE}" />`);
```

Comment:
I think can just modify `INDEX_MARKDOWN_DEFAULT` to include the new footer.

Reply:
Leftover from previous implementation that used `rootPath`, will refactor this.